zebra/zebra_ns.c

```diff
@@ -91,6 +91,7 @@ static int zebra_ns_delete(struct ns *ns)
 	if (!zns)
 		return 0;
 	XFREE(MTYPE_ZEBRA_NS, zns);
```

Comment:
Why not just free zns->info then?

Reply:
I dont understand. Im NULLing out the pointer that reference that points to the zns we just freed. (ns->info) points to this zns.

This is consistent with how we do it in `zebra/zebra_vrf` and the `vrf->info` pointer there.

Reply:
XFREE nulls out the pointer since it's a macro.  We should encourage this behavior instead of this pattern where we don't trust the results of the operatoin.

Reply:
Fixed